contrib/scalar/docs/philosophy.md

```diff
@@ -0,0 +1,71 @@
+The Philosophy of Scalar
+========================
+
+The team building Scalar has **opinions** about Git performance. Scalar
+takes out the guesswork by automatically configuring your Git repositories
+to take advantage of the latest and greatest features. It is difficult to
+say that these are the absolute best settings for every repository, but
+these settings do work for some of the largest repositories in the world.
+
+Scalar intends to do very little more than the standard Git client. We
+actively implement new features into Git instead of Scalar, then update
+Scalar only to configure those new settings. In particular, we are porting
+features like background maintenance to Git to make Scalar simpler and
+make Git more powerful.
+
+Scalar ships inside [a custom version of Git][microsoft-git], but we are
```

Comment:
```suggestion
Scalar ships inside [a custom version of Git](https://github.com/microsoft/git), but we are
```

Reply:
Oh, maybe that was intentional? I am not familiar with that sort of syntax...

Reply:
Yeah, it's a version of linking in markdown that can make the links look better in-line when writing. It's particularly helpful if the same link appears multiple times in the page, but can be confusing on the first read.

Reply:
TIL!